develop.sh

```diff
@@ -47,6 +58,15 @@ elif [ "$1" == "npm" ]; then
     npm_install
     exit
 
+elif [ "$1" == "spark" ]; then shift
+    if [ "$1" == 'format' ]; then
```

Comment:
```suggestion
    if [ "$2" == 'format' ]; then
```

Reply:
`shift` will move the pointer.

Reply:
Not shifting might be a better idea, it keeps the code easier to understand.

Reply:
Keeping uniformity in mind, I think we should retain `shift`  imo.

Reply:
Uniformity with what?

Reply:
With other functions in the script.

Reply:
The other functions only shift when they need to pass the rest of the args into a different command. I don't have stong opinions on this, happy to keep the shift here, but if I were writing this, I would do the following.

```
if "$1" == spark:
    if "$2" == format:
        format()
    else:
        shift 
        pass the remaining args to docker-compose
````

That way we only change the list when we actually need to change it.